core/src/io/github/eng12020team24/project1/MenuState.java

```diff
@@ -4,81 +4,80 @@
-    //private Stage menu;
     AuberGame game;
-    TextureRegion resumeButton;
     ActualGame actualGame = null;
-    TextureRegion playButton;
-    TextureRegion exitButton;
     TextureAtlas uiAtlas;
+    Button playButton;
+    Button exitButton;
+    Button resumeButton;
 
+    /**
+     * Initialises the Menu state
+     * @param game the overall {@link #AuberGame AuberGame} that this MenuState is a part of
+     */
     public MenuState(AuberGame game){
-        //menu = new Stage(new ScreenViewport());
         this.game=game;
-        //resumeButton = new Texture("uiElements/RESUME_BUTTON.png");
         uiAtlas = new TextureAtlas(Gdx.files.internal("UISpritesheet/uispritesheet.atlas"));
-        playButton=new TextureRegion(uiAtlas.findRegion("PLAY_BUTTON"));
-        resumeButton=new TextureRegion(uiAtlas.findRegion("RESUME_BUTTON"));
-        exitButton=new TextureRegion(uiAtlas.findRegion("EXIT_BUTTON"));
+        playButton = new Button(0, 128, uiAtlas.findRegion("PLAY_BUTTON"));
+        resumeButton = new Button(0, 128, uiAtlas.findRegion("RESUME_BUTTON"));
+        exitButton = new Button(0, 0, uiAtlas.findRegion("EXIT_BUTTON"));
     }
+
     @Override
 	public void show() {
-        //Table table = new Table();
-        //menu.addActor(table);
-    }
-    //@Override
-    public static void main(){
     }
+
     @Override
     public void pause(){
     }
+
+    /**
+     * Renders the menu screen - displays play/resume buttons depending on if the game has begun or not
+     * @param delta - necessary for render method however is unused
+     */
     @Override
     public void render(float delta){
-        Gdx.gl.glClearColor(1, 0, 0, 1);
+        Gdx.gl.glClearColor(0, 0, 0, 1);
 		Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
-        if (Gdx.input.getY()>Gdx.graphics.getHeight()-128 && Gdx.input.getX()< 450){ //exit
-            if (Gdx.input.isTouched()){
-                System.exit(0);
-            }
-        } else if (Gdx.input.getY() < Gdx.graphics.getHeight()-128 &&
-        Gdx.input.getY() > Gdx.graphics.getHeight() -256 &&
-        Gdx.input.getX() < 450){
-            if (Gdx.input.isTouched()){
-                if (this.actualGame == null) {
-                    this.actualGame = new ActualGame(game, this);
-                }
-                game.setScreen(actualGame);
-            }
+        if (exitButton.isClicked()){ //exits
+            Gdx.app.exit();
+        } else if (playButton.isClicked()){ // starts new game
+            this.actualGame = new ActualGame(game, this);
+            game.setScreen(actualGame);
+        } else if (resumeButton.isClicked()){ // resumes old game
+            game.setScreen(actualGame);
         }
+
         game.batch.begin();
-        game.batch.draw(exitButton,0,0);
-        if (this.actualGame == null) {
-            game.batch.draw(playButton,0,128);
+        game.batch.draw(exitButton.getTextureRegion(),exitButton.getX(),exitButton.getY());
```

Comment:
Why not have a Button.draw() method?

Reply:
because it needs to be drawn within the game batch, as far as I could tell